src/index.js

```diff
@@ -1,10 +1,33 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers'
+
 import './index.css';
-import App from './App';
+import App from './containers/App';
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+const generateID = () => {
+  return Math.floor((Math.random() * 1000) + 1);
+}
+
+const INITIAL_STATE = {
+  books: [
+    { id: generateID(), title: "Laws of Power", category: "Learning" },
+    { id: generateID(), title: "Why Men want Sex and Women want Love", category: "Learning" },
+    { id: generateID(), title: "Life 3.0", category: "Sci-Fi" }
+  ],
+}
+
+const store = createStore(rootReducer, INITIAL_STATE);
```

Comment:
It will be helpful for you guys to use the redux-devtool to be able to debug the store properly. I highly recommend it.